deepspeed/runtime/domino/transformer.py

```diff
@@ -249,6 +249,8 @@ def __init__(self,
                  output_bias=None):
         super(DominoTransformerLayer, self).__init__()
 
+        init_distributed()
```

Comment:
maybe change this dist init() like this:

https://github.com/microsoft/DeepSpeed/blob/da771ed42e41a44d5047813ca4672f1cfe9d1731/deepspeed/runtime/zero/mics.py#L150-L152